pkg/openapiclient/local_schemas_test.go

```diff
@@ -131,38 +90,32 @@ func Test_localSchemasClient_Paths(t *testing.T) {
 			"apis/storage.k8s.io/v1",
 		),
 	}, {
-		name: "invalid dir",
-		dir:  "invalid",
-		want: sets.New[string](),
+		name:    "invalid fs",
+		fs:      os.DirFS("../../invalid"),
+		want:    sets.New[string](),
+		wantErr: true,
 	}, {
-		name: "invalid fs",
-		fs:   os.DirFS("../../invalid"),
-		dir:  ".",
+		name: "not a dir",
+		fs:   os.DirFS("../../testcases/schemas/error_not_a_dir"),
 		want: sets.New[string](),
```

Comment:
This fails on windows returning a `nil` set rather than an empty set, but not on *nix.

It is also curious to me that we can return `nil` `Paths()` but no error

Reply:
> It is also curious to me that we can return nil Paths() but no error

If there's no `api` nor `apis` folders, what shall we return ?

Reply:
The problem was reading `/api` / `/apis` without trying to read `/` first 🙈